src/Tracker/trackerData.tsx

```diff
@@ -503,4 +485,230 @@ export const analyticsEventsMapping = new Map<
     'screen_welcome',
     { eventName: 'WELCOME', properties: { event_type: 'screen' } },
   ],
+
+  // TODO: Put in right place in this list
+  [
+    'screen_document_front_confirmation_cutoff_detected',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { cutoff: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_confirmation_cutoff_detected',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'back',
+        ui_alerts: { cutoff: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_front_confirmation_blur_detected',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { blur: 'error' },
```

Comment:
This could be a warning too, the user is only forced to retake the image on the first two attempts. The same applies to other image quality warnings.